modules/admin_manual/pages/configuration/server/occ_command/files_external_commands.adoc

```diff
@@ -68,6 +69,12 @@ be added as system mount.
 +----------+------------------+----------+
 ----
 
+NOTE: The `--importable-format` option helps make visible the technical mount settings.
```

Comment:
I am not a native speaker but this sounds better to me...
`helps make visible the technical mount settings.` --> `helps to make the technical mount settings visible.`

Reply:
I should read what I write! Fixed.